aten/src/ATen/native/transformers/attention.cpp

```diff
@@ -609,6 +610,15 @@ bool should_compute_logsumexp(const Tensor& query, const Tensor& key, const Tens
 
 } // namespace
 
+Tensor _safe_softmax(
+    const Tensor& self,
+    int64_t dim,
+    std::optional<ScalarType> dtype) {
+  auto out = at::softmax(self, dim, dtype);
+  const auto masked = self.eq(-std::numeric_limits<float>::infinity());
```

Comment:
@drisspg is it possible to put this float into a scalar tensor of dtype = torch.float32?